### DISCUSSION TITLE
Api  Hook auto create something on Directus System Collections

### DISCUSSION THREAD

**Discussion by okadts** [directus/directus#11637] [Archive]
Hi, any clue on how to add hook on Directus System Collections?
Like adding an user folder where they registered
Perhaps using helper, something simple like "directus.folder.create.... "
Looking for any example link
Thank you

**Comment by okadts** [comment]
@rijkvanzanten help please 🙏

**Comment by rijkvanzanten** [comment]
Take a look [at the available events](https://docs.directus.io/extensions/hooks/#available-events). There's CRUD hooks available for the system collections as well 👍🏻 

PS no need to tag me directly, I watch the whole repo and answer when I can 🙂
